pyproject.toml

```diff
@@ -99,7 +98,7 @@ docs = [
     "furo>=2023.09.10",
     "jupyter>=1.0.0",
     "jupytext>=1.16.1",
-    "myst-parser>=2.0.0",
+    "myst-parser>=4.0.0",
```

Comment:
Tried increasing the version of one of the docs parsers, let's see if this solves the issue with the weird link....

Reply:
Nope.